src/pages/Cafeteria/MobileCafeteriaPage/index.tsx

```diff
@@ -1,52 +1,77 @@
 import { cn } from '@bcsdlab/utils';
-import { CAFETERIA_CATEGORY, CAFETERIA_TIME } from 'static/cafeteria';
+import { useEffect, useState } from 'react';
+import { CAFETERIA_CATEGORY, MEAL_TYPES, MEAL_TYPE_MAP } from 'static/cafeteria';
+import { useDatePicker } from 'pages/Cafeteria/hooks/useDatePicker';
+import useCafeteriaList from 'pages/Cafeteria/hooks/useCafeteriaList';
+import { convertDateToSimpleString } from 'utils/ts/cafeteria';
 import useScrollToTop from 'utils/hooks/useScrollToTop';
-import { CafeteriaMenu } from 'interfaces/Cafeteria';
+import { MealType } from 'interfaces/Cafeteria';
+import useLogger from 'utils/hooks/useLogger';
 import WeeklyDatePicker from './components/WeeklyDatePicker';
 import MobileMenuBlock from './components/MobileMenuBlock';
 import styles from './MobileCafeteriaPage.module.scss';
 
 interface Props {
-  mealType: string;
-  setMealType: (mealType: string) => void;
-  cafeteriaList: CafeteriaMenu[];
-  useDatePicker: () => {
-    value: Date;
-    setPrev: () => void;
-    setNext: () => void;
-    setDate: (date: string) => void;
-  };
+  mealType: MealType;
+  setMealType: (mealType: MealType) => void;
 }
 
-export default function MobileCafeteriaPage({
-  mealType, setMealType, cafeteriaList, useDatePicker,
-}: Props) {
-  const {
-    value: currentDate,
-    setDate: onClickDate,
-  } = useDatePicker();
+export default function MobileCafeteriaPage({ mealType, setMealType }: Props) {
+  const logger = useLogger();
+  const [hasLoggedScroll, setHasLoggedScroll] = useState(false);
+  const { currentDate } = useDatePicker();
+  const { cafeteriaList } = useCafeteriaList(convertDateToSimpleString(currentDate));
+
+  const handleMealTypeChange = (meal: MealType) => {
+    logger.actionEventClick({ actionTitle: 'CAMPUS', title: 'menu_time', value: MEAL_TYPE_MAP[meal] });
+    setMealType(meal);
+  };
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const doc = document.documentElement;
+      const scrolled = doc.scrollTop;
+      const maxHeight = doc.scrollHeight - doc.clientHeight;
+      const scrollPercentage = (scrolled / maxHeight) * 100;
+
+      if (scrollPercentage > 70 && !hasLoggedScroll) {
+        logger.actionEventClick({ actionTitle: 'SCROLL', title: 'menu_time', value: MEAL_TYPE_MAP[mealType] });
```

Comment:
`actionTitle`에는 팀명이 들어가야됩니다!
```suggestion
        logger.actionEventClick({ actionTitle: 'CAMPUS', title: 'menu_time', value: MEAL_TYPE_MAP[mealType] });
```

Reply:
오 실수했네요